substrate/frame/benchmarking/src/v1.rs

```diff
@@ -861,7 +861,7 @@ macro_rules! impl_bench_name_tests {
 					// Same per-case logic as when all cases are run in the
 					// same function.
 					match std::panic::catch_unwind(|| {
-						$module::<$test>::[< test_benchmark_ $name >] ()
+						$module::<$test>::test_bench_by_name(stringify!($name).as_bytes())
```

Comment:
Why was this changed?

Reply:
This is to ensure this generated `#[test]` calls can be executed for both `v1` and `v2` produced benchmarks, as `v1` produces calls named `Pallet::test_bench_$benchmark_name`, while `v2` produces calls named `test_{benchmark_name}`, but both macros share the `test_bench_by_name` call.

Another possibility is to achieve this, while ensuring retrocompatibility with `v1` is to change its analog benchmark generation in `v2` macro, like this:

> In `substrate/frame/support/procedural/src/benchmark.rs` line `434`:
```patch
- 		.map(|n| Ident::new(format!("test_{}", n).as_str(), Span::call_site()))
+ 		.map(|n| Ident::new(format!("test_benchmark_{}", n).as_str(), Span::call_site()))
```

Reply:
Yes i think that is better. I would like to keep the changes to the old V1 syntax as minimal as possible - since dozens f projects still use it and possibly depend on implementation specific behaviour.